client/src/ui/BitbakeStatusBar.ts

```diff
@@ -44,13 +45,15 @@ export class BitbakeStatusBar {
     })
 
     vscode.tasks.onDidStartTask((e: vscode.TaskStartEvent) => {
-      if (e.execution.task.name === 'Bitbake: Parse' && e.execution.task.source === 'bitbake') {
+      const taskName = e.execution.task.name
+      if ((taskName === 'Bitbake: Parse' || taskName === BitbakeRecipeScanner.taskName) && e.execution.task.source === 'bitbake') {
```

Comment:
There is a subtle bug. If both a recipe scan and a global parse are started together, we lose the "scanning" status bar which goes into "building". To fix this, we can have separate this.parsingInProgress, this.recipeScanInProgress.

Reply:
Done. The error status can show as expected as far as I can see.